### ISSUE TITLE
Feature request: easy atomic swaps in the GUI

### ISSUE THREAD

**Issue by jeroz1** [RavenProject/Ravencoin#423]
tl;dr As atomic swaps are possible (https://github.com/RavenProject/Ravencoin/blob/master/doc/atomicswaps.md), how feasible is it to put this feature in the GUI?
__________
In Ravenville, little Lisa is building her own lemonade stand and distributed lemonade-stand-shares to the neighbors in the form of LEMONADE tokens. Her uncle in Bird City, would also like to chip in. In the past, he encountered some problems with sending money and he wants to make sure that it is fast, easy and secure using RVN. It would be convenient if Lisa could send her uncle a request for a secure transaction using an atomic swap request of 10 RVN for 2 LEMONADE.
    After a hot summer, Lisa had a very profitable season, she sent out dividends to her shareholders and her mother decides that she wants to quickly sell some LEMONADE tokens while uncle John wants to double down for more profits next year. They plan on making an OTC trade as they don't feel like going trough the hassle of an exchange. So they want to make an atomic swap too. 
    Lisa's little brother, Milo, learned a great deal from his bigger sister and decides to set up a cookie stand using Lisa's business model and he started issuing COOKIE tokens. Lisa also likes to invest but just bought inventory to sell hot chocolate as winter is coming (pun intended). She does, however, have LEMONADE tokens that are worth at least 5 COOKIE. They therefore would like to securely swap the tokens between each other. 
    For her hot chocolate idea, Lisa wants to set up a campaign to let everyone know they can get hot chocolate this winter at the lemonade stand. For her promotion, she wants to give all neighbors a DRINK token that they can redeem for a free hot chocolate drink. Unfortunately, the name DRINK is already taken by her classmate Mary. Mary, knows about Lisa her lemonade stand from school presentations, and likes to get some tokens too. So, they want to make a secure easy and quick transfer to swap the asset DRINK for 10 LEMONADE tokens. 

How feasible is it to put an atomic swap feature in Raven Core? Allowing for sending/receiving easy, secure, and fast transactions, such as (non-scammable) OTC trades:
- STO's for RVN or other tokens,
- Asset names for RVN or other tokens,
- Game currency for unique legendary swords, 
- etc.

**Comment by ypey** [comment]
extremely usefull, please add this feature to the GUI!..... crypto is in need of more userfriendlyness overall
